pkg/sql/schemachanger/scexec/scmutationexec/dependencies.go

```diff
@@ -115,4 +116,10 @@ type DeferredMutationStateUpdater interface {
 
 	// RefreshStats refresh stats for a given descriptor.
 	RefreshStats(id descpb.ID)
+
+	// AddIndexForMaybeSplitAndScatter adds an index to maybe split and scatter,
+	// if it makes sense.
```

Comment:
reword comment

Reply:
Reworded to: 	// AddIndexForMaybeSplitAndScatter splits and scatters rows for a given index,
	                // if it's either hash sharded or under the system tenant.